node/Dockerfile

```diff
@@ -2,13 +2,6 @@
 FROM --platform=linux/amd64 docker.io/golang:1.20.10-bullseye@sha256:082569b3303b164cc4a7c88ac59b19b69c1a5d662041ac0dca046ac239632442 AS build
 # libwasmvm.so is not compatible with arm
 
-# Support additional root CAs
-COPY node/go.mod cert.pem* /certs/
```

Comment:
looks like `node/go.mod` here was not actually needed, no command references it and line 19 below copies it again anyway, so 👍